packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -536,6 +542,9 @@ class ThemeData extends Diagnosticable {
   /// Configures the hit test size of certain Material widgets.
   final MaterialTapTargetSize materialTapTargetSize;
 
+  /// Default [PageRoute] transitions per [TargetPlatform].
```

Comment:
Not quite sure what "per [TargetPlatform]" means here. Maybe elaborate a bit for these docs?